### PR TITLE
fix azure-data-confidentialledger readme issues

### PR DESCRIPTION
**Pull request by LuHang2021**
Fix issue https://github.com/Azure/azure-sdk-for-java/issues/21958
@scottaddie , @joshfree, @lmazuel , @christothes and @jongio for notification

### CODE REVIEW

File: sdk/confidentialledger/azure-security-confidentialledger/README.md [hunk 1]
```diff
@@ -150,6 +150,15 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
 [package]: https://mvnrepository.com/artifact/com.azure/azure-security-confidentialledger
 [samples_readme]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/confidentialledger/azure-security-confidentialledger/src/samples/README.md
+[low_level_client]: https://github.com/Azure/azure-sdk-for-java/wiki/Low-Level-Client
+[azure_resource_manager]: https://docs.microsoft.com/azure/azure-resource-manager/
+[azure_identity]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity
```
Comment (joshfree) [inline sdk/confidentialledger/azure-security-confidentialledger/README.md:155]: replace master with main

File: sdk/confidentialledger/azure-security-confidentialledger/README.md [hunk 2]
```diff
@@ -150,6 +150,15 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
...
+[DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/identity/azure-identity/README.md#defaultazurecredential
```
Comment (joshfree) [inline sdk/confidentialledger/azure-security-confidentialledger/README.md:156]: replace master with main

**Review by joshfree — CHANGES_REQUESTED** [review]

File: sdk/confidentialledger/azure-security-confidentialledger/README.md (hunk 2 above)
Comment (LuHang2021) [reply sdk/confidentialledger/azure-security-confidentialledger/README.md]: I have updated.

File: sdk/confidentialledger/azure-security-confidentialledger/README.md (hunk 1 above)
Comment (LuHang2021) [reply sdk/confidentialledger/azure-security-confidentialledger/README.md]: I have updated.

**Comment by v-xuto** [comment]
@joshfree - We have resolved the comments, please re-review.

File: sdk/confidentialledger/azure-security-confidentialledger/README.md [hunk 3]
```diff
@@ -150,6 +150,15 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
 [package]: https://mvnrepository.com/artifact/com.azure/azure-security-confidentialledger
 [samples_readme]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/confidentialledger/azure-security-confidentialledger/src/samples/README.md
+[low_level_client]: https://github.com/Azure/azure-sdk-for-java/wiki/Low-Level-Client
```
Comment (JonathanGiles) [inline sdk/confidentialledger/azure-security-confidentialledger/README.md:153]: This page does not exist at present.

Comment (hongli750210) [reply sdk/confidentialledger/azure-security-confidentialledger/README.md]: The `Low-Level client` in track2 has been abandoned. so we think it is more appropriate to modify the description of `Low-Level client` in the document to `Http client`. Do you think this is okay?

**Comment by v-xuto** [comment]
@JonathanGiles  We have updated by your comments, please re-review.

File: sdk/confidentialledger/azure-security-confidentialledger/README.md (hunk 3 above)
Comment (hongli750210) [reply sdk/confidentialledger/azure-security-confidentialledger/README.md]: The description related to `HTTP client` has been removed.

**Comment by v-xuto** [comment]
@JonathanGiles We have removed HTTP client link, please review this PR.

**Comment by v-xuto** [comment]
@JonathanGiles Please review this PR when you are free.

**Comment by JonathanGiles** [comment]
Do you have a link to the javadocs that you can provide?

**Comment by hongli750210** [comment]
@JonathanGiles  I think javadocs refers to the link on the picture 
image
, so there is no need to add a link to the javadoc.

**Comment by JonathanGiles** [comment]
Which one of is the Javadocs that the user should click on?

**Comment by jongio** [comment]
@lmazuel and @christothes - Could you please have a look at the issue here and review the PR to help close on this?
https://github.com/Azure/azure-sdk-for-java/issues/21958

Thanks,
Jon

File: sdk/confidentialledger/azure-security-confidentialledger/README.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 Azure Confidential Ledger provides a service for logging to an immutable, tamper-proof ledger. As part of the [Azure Confidential Computing][azure_confidential_computing]
 portfolio, Azure Confidential Ledger runs in SGX enclaves. It is built on Microsoft Research's [Confidential Consortium Framework][ccf].
 
-**Please rely heavily on the javadocs and our [Low-Level client docs][low_level_client] to use this library**
+**Please rely heavily on the javadocs to use this library**
```
Comment (christothes) [inline sdk/confidentialledger/azure-security-confidentialledger/README.md:6]: Let's just remove this line for now, unless we have links to some specific javadocs we can point to.

Comment (JonathanGiles) [reply sdk/confidentialledger/azure-security-confidentialledger/README.md:6]: Agreed

Comment (hongli750210) [reply sdk/confidentialledger/azure-security-confidentialledger/README.md]: Okay, this line has been removed for now.

**Review by christothes — APPROVED** [review]

**Comment by christothes** [comment]
@joshfree Are you good to sign off? Looks like your vote is blocking.

**Pull request merged**